src/django_components/commands/upgrade.py

```diff
@@ -0,0 +1,79 @@
+import os
+import re
+from pathlib import Path
+from typing import Any
+
+from django.conf import settings
+from django.template.engine import Engine
+
+from django_components.template_loader import Loader
+from django_components.util.command import CommandArg, ComponentCommand
+
+
+# TODO_V1 - Remove, no longer needed?
+class UpgradeCommand(ComponentCommand):
```

Comment:
`upgradecomponent` is the same as `startcomponent`, that the original API was kept for backwards compatibility. But the command itself is now defined as a `ComponentCommand` subclass, so it's independent of Django. And the new way to call this command is `components upgrade`.